components/feature/top-sites/src/androidTest/java/mozilla/components/feature/top/sites/OnDevicePinnedSitesStorageTest.kt

```diff
@@ -28,7 +28,7 @@ import java.util.concurrent.Executors
 private const val MIGRATION_TEST_DB = "migration-test"
 
 @Suppress("LargeClass")
-class PinnedSitesStorageTest {
+class OnDevicePinnedSitesStorageTest {
```

Comment:
I renamed this because now there is also a _regular_ test. And in most cases where a class is tested in both `test` and `androidTest`, the test in `androidTest` gets prefixed with `OnDevice`. I found these examples:

`components/browser/icons/src/androidTest/java/mozilla/components/browser/icons/OnDeviceBrowserIconsTest.kt`
`components/browser/icons/src/test/java/mozilla/components/browser/icons/BrowserIconsTest.kt`

`components/support/images/src/androidTest/java/mozilla/components/support/images/decoder/OnDeviceAndroidImageDecoderTest.kt`
`components/support/images/src/test/java/mozilla/components/support/images/decoder/AndroidImageDecoderTest.kt`

`components/feature/sitepermissions/src/androidTest/java/mozilla/components/feature/sitepermissions/db/OnDeviceSitePermissionsStorageTest.kt`
`components/feature/sitepermissions/src/test/java/mozilla/components/feature/sitepermissions/SitePermissionsStorageTest.kt`

`components/feature/prompts/src/androidTest/java/mozilla/components/feature/prompts/file/OnDeviceFilePickerTest.kt`
`components/feature/prompts/src/test/java/mozilla/components/feature/prompts/file/FilePickerTest.kt`

`components/feature/downloads/src/androidTest/java/mozilla/components/feature/downloads/OnDeviceDownloadStorageTest.kt`
`components/feature/downloads/src/test/java/mozilla/components/feature/downloads/DownloadStorageTest.kt`

And only one example to the contrary:

`components/browser/icons/src/androidTest/java/mozilla/components/browser/icons/decoder/ICOIconDecoderTest.kt`
`components/browser/icons/src/test/java/mozilla/components/browser/icons/decoder/ICOIconDecoderTest.kt`

Of course I will happily oblige and not rename this class. But I want to make sure you know my reasons for proposing to rename it. Please let me know.

Reply:
Gotcha, I think this is good. Thanks for the details!